docs/programme/livestream/scenes.md

```diff
@@ -4,6 +4,14 @@ This page documents a few of the core scenes which are used for livestreams. Not
 
 The various assets useful for creating these scenes can be found [here](https://drive.google.com/drive/folders/1pr1vKpnonxFoO8O5chsH5DF7Vsd0Tfih?usp=sharing).
 
+## Presentation
+
+Events such as Kickstart are predominantly a presentation.
+
+For the presentation, it's recommended to use a window-based capture, rather than screen-based, to reduce the risk of accidentally putting sensitive or unrelated content over the slides.
+
+Long presentations should still show the host. Choose a bottom corner of the screen which won't cover any content and keep their camera fixed there. Slides may need modifying to ensure important content isn't covered.
+
```

Comment:
May be worth adding a section here on audio, namely not using global audio sources 
![image](https://github.com/srobo/runbook/assets/15033502/ff69e09c-2e12-43a6-b5e2-62024439b2a6)
(the only exception _maybe_ being microphones as we'll have an interface to mute channels anyway) and individually adding sources in OBS. This means desktop audio won't come through on the stream. @ZachWilliamsSR last year showed me this tool (https://vb-audio.com/Voicemeeter/) to allow you to create "virtual" audio sources to pipe applications not via the main speaker. 

In Windows 11 in the mixer option you can also choose which output device to send audio to:
![image](https://github.com/srobo/runbook/assets/15033502/32b983c7-16bf-4c71-9e46-f8bf431055b6)

Reply:
Great idea, will add!